sqflite/android/build.gradle

```diff
@@ -35,6 +35,7 @@ android {
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles 'proguard-rules.txt'
```

Comment:
I you comment out this line you will see the problematic `MissingPluginException(No implementation found for method getDatabasesPath on channel com.tekartik.sqflite)` exception. With the Proguard rules everything should work.